src/cartesiangrid.jl

```diff
@@ -161,14 +159,13 @@ function vizgrid!(plot)
   viz!(plot, mesh,
     color = color,
     alpha = alpha,
-    showfacets = false
+    aes = Aes(segments=false)
```

Comment:
We need forward other `Aes` attributes?

Reply:
For now let's just make the minimum changes necessary. We can think about it later.